packages/core/src/components/ui/Checkbox/Checkbox.js

```diff
@@ -123,6 +127,10 @@ Checkbox.propTypes = {
    * If true, shows the Button in a small style.
    */
   small: PropTypes.bool,
+  /**
+   * A help message to be shown in the tooltip.
+   */
+  customhelp: PropTypes.string,
```

Comment:
can you rename customhelp to helpMessage, custom word does not seems to make sense to me